content/strategy-goals/strategy/cloud/growth-team/index.md

```diff
@@ -1,47 +1,68 @@
 
-TODO
+- We know our product is incredibly appealing and sticky to users once they understand what it does, know how to use it, and have figured out how to integrate it into their workflows. We also know **users struggle to learn how the product works** and to understand exactly how it will help them solve critical problems.
+- We see sharp drop off in the funnel of website visitors -> sign ups / active trials. Current user flows **do not gather contact information early enough** nor do they obtain the necessary **legal clearance** (acceptance of terms & privacy policy) to allow product and marketing to re-engage website visitors.
+- We **lack comprehensive onboarding in the self-hosted product**. Ensuring that allocated licenses have high rates of usage is critical in driving renewal and expansion within our enterprise customers.
+- Product owned and marketing owned **web pages do not feel like a single cohesive web experience**. More unified language and user flows are needed to drive each user persona to the optimal assets and experiences that will help them understand the product and move toward conversion.
 
 ## Strategy and Plans
 
-TODO
-
-### Goals
+The growth team’s focus is to increase activation of two distinct groups: website visitors and new enterprise license holders. Sourcegraph’s product is powerful, but also complex and technical. Helping new users understand what Sourcegraph can do and developing new habits to incorporate it into their workflow will require a robust and finely tuned activation journey. In order to increase activation the growth team will focus on several key areas:
 
-TODO
+- Increasing _registrations_ of website visitors by removing friction in the registration process and clearer CTA in the product experience outside the login wall. Registrations provide key information and fulfill a critical legal requirement to unlock opportunities to nurture users from early in their journey through activation.
+- Improve _retention & engagement_ through early onboarding to the product. Both new website visitors and new customers will need personalized guided product experiences that help them reach the aha moment where they understand the value of the product and have it integrated into their environment and workflows. We will partner with existing customers and enterprise users to discover the most optimal way to onboard to SG.[^1]
+- Improve _cohesion_ between product and marketing/sales language by aligning messaging on sourcegraph.com with the five use cases. Customer feedback and traction in sales have shown the use cases resonate with our customers. Though the language around the use cases is apparent in the newly refreshed marketing site, these value stories can be better told throughout the product experience - particularly for new users.
 
-### Themes
+### Goals
 
-TODO
+- Improve visitor registration rate (# of visitors who sign up / # of users visiting over the period)
+- Reduce 1 and done rate
```

Comment:
I'm not sure if I fully understand this. Can you, please, elaborate on what this means (maybe some examples)?

P.S: I guess it is about doing some action once and never again. But still not sure what kind of actions/things we mean in these cases.